core/src/main/scala/org/apache/spark/util/RpcUtils.scala

```diff
@@ -54,6 +56,14 @@ private[spark] object RpcUtils {
     RpcTimeout(conf, Seq(RPC_LOOKUP_TIMEOUT.key, NETWORK_TIMEOUT.key), "120s")
   }
 
+  /**
+   * Infinite timeout is used internally, so there's no timeout configuration property that
+   * controls it. Therefore, we use "infinite" without any specific reason as its timeout
+   * configuration property. And its timeout property should never be accessed since infinite
+   * means we never timeout.
+   */
+  val infiniteTimeout = new RpcTimeout(Long.MaxValue.nanos, "infinite")
```

Comment:
shall we upper case the name as it's a constant?

Reply:
sure, updated to `INFINITE_TIMEOUT`